app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -450,7 +450,7 @@ class Exposure extends React.Component {
               className="pt-2 my-auto"
               type="switch"
               id="was_in_health_care_facility_with_known_cases"
-              label="WAS IN HEALTH CARE FACILITY WITH KNOWN CASES"
+              label="WAS IN HEALTHCARE FACILITY WITH KNOWN CASES"
```

Comment:
Out of curiosity, what prompted this change? Some quick searching makes me think there is little consensus over 1 word vs. 2.

Reply:
Because everywhere else in that file had it as one word (healthcare personnel and facility) and the different uses in `Exposure.js` and `Patient.js` files were causing some issues with the system tests because of the lack of consistency.

Reply:
Although I am now noticing that this database field is `health_care` versus the other ones have no _ between health and care